docs/src/ex/0_Introduction.md

```diff
@@ -0,0 +1,178 @@
+    plot(x, pdf.(Ref(updated_belief), x),
+        size = (500, 250),
+        title = "Updated belief after $N observations",
+        xlabel = "probability of heads",
+        ylabel = "",
+        legend = nothing,
+        xlim = (0,1),
+        fill=0, α=0.3, w=3)
+    vline!([p_true])
+end;
+```
+
+![animation](https://user-images.githubusercontent.com/7974003/44995702-37c1b200-af9c-11e8-8b26-c88a528956af.gif)
+
+The animation above shows that with increasing evidence our belief about the probability of heads in a coin flip slowly adjusts towards the true value. The orange line in the animation represents the true probability of seeing heads on a single coin flip, while the mode of the distribution shows what the model believes the probability of a heads is given the evidence it has seen.
+
+## Coin Flipping with Turing
+
+In the previous example, we used the fact that our prior distribution is a [conjugate prior](https://en.wikipedia.org/wiki/Conjugate_prior). Note that a closed-form expression (the `updated_belief` expression) for the posterior is not accessible in general and usually does not exist for more interesting models.
+
+We are now going to move away from the closed-form expression above and specify the same model using Turing*. To do so, we will first need to import `Turing`, `MCMCChain`, `Distributions`, and `StatPlots`. `MCMChain` is a library built by the Turing team to help summarize Markov Chain Monte Carlo (MCMC) simulations, as well as a variety of utility functions for diagnostics and visualizations.
+
+```julia
+# load Turing and MCMCChain
+using Turing, MCMCChain
+
+# load the distributions library
+using Distributions
+
+# load stats plots for density plots
+using StatPlots
+```
+
+First, we will define the coin-flip model using Turing.
+
+```julia
+@model coinflip(y) = begin
+
+    # our prior belief about the probability of heads in a coin
+    p ~ Beta(1, 1)
+
+    # the number of observations
+    N = length(y)
+    for n in 1:N
+        # heads or tails of a coin are drawn from a Bernoulli distribution
+        y[n] ~ Bernoulli(p)
+    end
+end;
+```
+
+After defining the model, we can approximate the posterior distribution by pulling samples from the distribution. In this example, we use a [Hamiltonian Monte Carlo](https://en.wikipedia.org/wiki/Hamiltonian_Monte_Carlo) sampler to construct these samples. Later tutorials will give more information on the samplers available in Turing and discuss their use for different models.
+
+```julia
+# setting of Hamiltonian Monte Carlo (HMC) sampler
+iterations = 1000
+ϵ = 0.05
+τ = 10
+
+# start sampling
+chain = sample(coinflip(data), HMC(iterations, ϵ, τ));
+```
+
+After finishing the sampling process, we can visualize the posterior distribution approximated using Turing against the posterior distribution in closed-form. We can extract the chain data from the sampler using the `Chains(chain[:p])` function. This contains all the values of `p` we drew while sampling.
```

Comment:
`Chain(chain[:p])` creates an instance of the Chain type which summarizes the MCMC simulation.

Reply:
I have amended the paragraph to read the following, let me know what you think of this:

>After finishing the sampling process, we can visualize the posterior distribution approximated using Turing against the posterior distribution in closed-form. We can extract the chain data from the sampler using the `Chains(chain[:p])` function, exported from the `MCMCChain` module. `Chains(chain[:p])` creates an instance of the `Chain` type which summarizes the MCMC simulation -- the `MCMCChain` module supports numerous tools for plotting, summarizing, and describing variables of type `Chain`.